zerver/lib/zcommand.py

```diff
@@ -1,12 +1,16 @@
 from typing import Any, Dict
 from django.utils.translation import ugettext as _
 
-from zerver.models import UserProfile
+from zerver.models import UserProfile, get_realm_stream, get_user, Stream
 from zerver.lib.actions import do_set_user_display_setting
 from zerver.lib.exceptions import JsonableError
+from zerver.lib.bugdown import possible_linked_stream_names, MentionData
 
 def process_zcommands(content: str, user_profile: UserProfile) -> Dict[str, Any]:
-    command = content.replace('/', '')
+    content = content[1:]
+
+    tokens = content.split(' ')
```

Comment:
Is this going to work if there are spaces in the stream name or user name?

Reply:
Yes it will. I have added a test for this [here](https://github.com/rheaparekh/zulip/blob/e523352f4b4424800ee4107904ce1a5deade8dcc/zerver/tests/test_zcommand.py#L78).

Reply:
This was the minor overlap/confusion re testing for spaces in the `/mute_topic` PR.